test/algorithms/is_valid.cpp

```diff
@@ -238,7 +238,7 @@ BOOST_AUTO_TEST_CASE( test_is_valid_multilinestring )
 }
 
 template <typename Point, bool AllowDuplicates>
-void test_open_rings()
+inline void test_open_rings()
```

Comment:
In the past you had "complained" about it, so I decided to consistently use `inline` even in `.cpp` files.